cms/templates/course_outline.html

```diff
@@ -132,6 +132,13 @@ <h3 class="title-3">${_("Changing the content students see")}</h3>
             </div>
 
         </aside>
+        % if settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False) and context_course.licensable and context_course.license:
+          <div class="course-license">
+              <div class="xmodule_licensable license">
+                ${context_course.license.html}
```

Comment:
You should not have a method generating HTML that you can dump into the template like this. The HTML should be generated in the template. That's what templates are for.